src/microbe_stage/MicrobeAI.cs

```diff
@@ -305,8 +305,15 @@ private void PursueAndConsumeChunks(Vector3 chunk, Random random)
             MoveWithRandomTurn(0.1f, 0.2f, random);
         }
 
-        // Always set target Position, for use later in AI
-        microbe.MovementDirection = new Vector3(0.0f, 0.0f, -Constants.AI_BASE_MOVEMENT);
+        // If this Microbe is right on top of the chunk, stop instead of spinning
+        if (DistanceFromMe(chunk) < microbe.Radius)
```

Comment:
I think `microbe.Radius` should be replaced with a constant defined in Constants, something like `MICROBE_ENGULF_CHUNK_STOP_DISTANCE = 0.8f`
My reasoning is that as most chunks have radius of 1, that's mostly fine, but single hex bacteria, will have a radious of 0.5, I'm a little scared that those cells might stop too soon, and 0.8 probably is low enough that the cells do get within that range during an AI update and can stop moving.
Other than that I'd like to see playtesting on this PR to ensure the AI no longer is seen spinning around.

Reply:
True, but keep in mind that a microbe can't engulf anything until it is at least engulf size 2. That said, with a fixed stop distance I think any positive value will be the same, landing on the chunk and flipping once.